assets/src/edit-story/components/canvas/singleSelectionMoveable/useDrag.js

```diff
@@ -57,6 +58,9 @@ function useSingleSelectionDrag({
   );
 
   const onDrag = ({ target, beforeTranslate, clientX, clientY }) => {
+    if (!isDragging) {
+      setIsDragging(true);
+    }
```

Comment:
I don't think the `if` is necessary here. If the `useState` is set to the same value it already has, nothing happens, so I don't think there's a problem with setting it to `true` every time it's moving around.

But it would be nice to make sure, that we don't get unnecessary re-renders, if this guard is removed.

Reply:
Looks like all good without it, will remove and send to QA, thanks!